recipes/pysbol/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pysbol" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SynBioDex/pySBOL/archive/{{ version }}.tar.gz
+  sha256: 153432071fdba11ad3612c183f589bee8d9855d088203d5be8832c8026c4e5ec
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+
+requirements:
+  host:
+    - python =3.7
+    - pip
+  run:
+    - python =3.7
+    - libxslt
```

Comment:
I don't see runtime requirements on `requests` or `libxslt` in the `setup.py`.

Reply:
Hi @synapticarbors ,

we only need **requests** dependency. Thank you.